docs/cloud/faq/roadmap.md

```diff
@@ -0,0 +1,27 @@
+# Roadmap
+
+## Migrating to Cloud
+
+### What is the ETA on the tool being developed to assist in migration of existing customer data?
+
+We have a replicator that is available now. 
+Depending on your requirements this may be suitable
```

Comment:
```suggestion
Depending on your requirements it may be suitable.
```

Reply:
Might be worth elaborate or link to the docs. This sounds a bit enigmatic.
@ylorph @alexeyzimarev, can we link the Replicator tool page here?